contracts/contracts/RequestManager.sol

```diff
@@ -391,4 +392,9 @@ contract RequestManager is Ownable {
         gasPrice = newGasPrice;
         serviceFeePPM = newServiceFeePPM;
     }
+
+    function setFinalizationTime(uint256 targetChainId, uint256 finalizationTime) external onlyOwner {
+        require(finalizationTime > 0, "Finalization time of rollup must be greater than 0");
```

Comment:
Maybe something like "Rollup is not registered" is better? The current error is not really actionable for the user.

Reply:
Isn't the only user the owner of the contract, hence we?